mycroft/configuration/defaults/defaults.ini

```diff
@@ -92,3 +92,7 @@ max_time = 600  # in seconds
 notify_delay = 5  # in seconds
 rate = 16000
 channels = 1
+
+[NestSkill]
+nest_user = "default-user"
+nest_password = "default_password"
```

Comment:
Since those variables are inside NestSkill session there is no real need to use the 'nest' prefix. Then we can simply use `user` and `password` (even `pass`). Suggestion:
```
[NestSkill]
user = "default-user"
password = "default_password"
```